packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1274,4 +1274,46 @@
     <protected>false</protected>
     <shortDescription>Search instead for</shortDescription>
   </labels>
+  <labels>
```

Comment:
Helo @cmartel-coveo! 

Can you please help me review the following labels that will be used inside the new feedback modal implemented for the RGA feature in Quantic. 
Only what's inside the tag `<value>...</value> is important.
Thank you!

Reply:
@mmitiche @SimonMilord Let me know if I forgot something. Looking through github isn't very... natural for me 😅